src/plan.rs

```diff
@@ -668,8 +697,15 @@ mod unit_tests {
     fn test_scalar_recipe_cache() {
         // Check that all butterflies are used
         let mut planner = FftPlannerScalar::<f64>::new();
-        let fft_a = planner.design_fft_for_len(1234);
-        let fft_b = planner.design_fft_for_len(1234);
-        assert!(Rc::ptr_eq(&fft_a, &fft_b), "Existing recipe was not reused");
+        let fft_a: RecipeId = planner.design_fft_for_len(1234);
+        let fft_b: RecipeId = planner.design_fft_for_len(1234);
+        assert_eq!(fft_a, fft_b, "Existing recipe was not reused");
+    }
+
+    fn test_impl_fft_planner_traits<T: FftNum>() {
```

Comment:
Here is the `FftPlanner: impl Send` test. (Note: I guess I should add a #[test] to avoid the dead code warnings.)